cmd/kube-scheduler/app/server.go

```diff
@@ -133,13 +133,12 @@ for more information about scheduling and the kube-scheduler component.`,
 // runCommand runs the scheduler.
 func runCommand(cmd *cobra.Command, opts *options.Options, registryOptions ...Option) error {
 	verflag.PrintAndExitIfRequested()
-	fg := opts.ComponentGlobalsRegistry.FeatureGateFor(utilversion.DefaultKubeComponent)
 	// Activate logging as soon as possible, after that
 	// show flags with the final logging configuration.
-	if err := logsapi.ValidateAndApply(opts.Logs, fg); err != nil {
```

Comment:
I initially changed it for consistency within the codebase by directly using 'opts.ComponentGlobalsRegistry.FeatureGateFor(utilversion.DefaultKubeComponent)' rather than introducing an additional variable 'fg'. 
However, I now understand that the original code was cleaner, so I will revert this part.